packages/notifications/src/Http/Livewire/Notifications.php

```diff
@@ -83,7 +83,12 @@ public function getDatabaseNotifications(): DatabaseNotificationCollection
     public function getDatabaseNotificationsQuery(): Builder | Relation
     {
         /** @phpstan-ignore-next-line */
-        return $this->getUser()->notifications()->where('data->format', 'filament');
+        return match ($this->getUser()->notifications()->getConnection()->getDriverName()) {
```

Comment:
Could you extract `$this->getUser()->notifications()` to `$query` just above the match expression?

Reply:
Mark as ready for review when done please.